Configuration/Geometry/README.md

```diff
@@ -14,7 +14,7 @@ Calorimeters:
 * C6: HGCal (v9) + HFNose + Phase2 HCAL and EB
 * C8: HGCal (v10 post TDR HGCal Geometry) + Phase2 HCAL and EB + Tracker cables in calorimeter region
 * C9: HGCal (v11 post TDR HGCal Geometry same as C8 + corner centering for HE part)
-* C10: HGCal (as in C9) + HFNose + Phase2 HCAL and EB
+* C10: HGCal (as in C6 but with corrected wafer size) + HFNose with correted wafer size + Phase2 HCAL and EB
```

Comment:
@bsunanda sorry, but I don't understand this change at all. I thought the point of this PR was to propagate the fixed HGCal v11 wafer size to D47. Now the whole D47 workflow has reverted back to HGCal v9? Why?

Reply:
This is what Mariarosaria wanted. She wants V9 like HGCal (i.e. without the support structure but with correct wafer size). Have to update HGCal support structure sometime soon because there has been an update and which will be less prohibitive to HFNose. Corrected wafer size will affect the corverage somewhat and also partial wafer definition.

Reply:
@bsunanda okay, this was not clear from the PR description (which should be updated with these details).

Does this mean that there is no longer any desire for the existing D47 with HGCal v11? I'm worried now that this will cause confusion.

Reply:
For the D47 we should have
- HFNOSE: with V9 but corrected wafer size
- HGC with the latest and greatest things available (i.e. V11 corrected and support structure as in Febrary 2019).